tests/unit/test_report_builder.py

```diff
@@ -28,7 +28,7 @@ def before_listener():
     before_fired = True
 
 
-def after_listener(static_folder, output_folder) -> bool:
+def after_listener(static_folder, output_folder, context) -> bool:
     global after_fired
```

Comment:
avoid using global variables; is there another way of doing this?

Reply:
I rewrote that.